test/ansible/roles/bmh_setup/tasks/main.yaml

```diff
@@ -0,0 +1,50 @@
+---
+# create libvirt BMH network with dns server
+- name: create libvirt network
+  shell: export LOADBALANCER_IP=$(cat /tmp/loadbalancer_ip) && cat "{{ src_dir}}/test/e2e/libvirt/network-bmh.xml.tpl" | sed "s/<LOADBALANCER_IP>/${LOADBALANCER_IP}/g" >/tmp/libvirt-network-bmh.xml && (virsh net-destroy bmh || true) && (virsh net-undefine bmh || true) && virsh net-define /tmp/libvirt-network-bmh.xml
+- name: start libvirt network
+  shell: virsh net-start bmh
+
+  # create VMs
+- name: Copy VM creation script
+  copy:
+    dest: "/tmp/vm_functions"
+    content: |
+      #!/bin/bash
+      function create_vm {
+        name=$1
+        mac=$2
+        virsh destroy "${name}" 2>/dev/null || true
+        virsh undefine --domain "${name}" --remove-all-storage --nvram 2>/dev/null || true
+        virt-install -n "${name}" --pxe --os-variant=rhel8.0 --ram=16384 --vcpus=8 --network network=bmh,mac="${mac}" --disk size=120,bus=scsi,sparse=yes --check disk_size=off --noautoconsole
+      }
```

Comment:
I think `ansible-lint` would complain because it would expect to set permissions on the file